src/paint/paint.h

```diff
@@ -24,6 +24,12 @@
 
 typedef struct attached_paint_struct attached_paint_struct;
 
+#ifdef NO_RCT2
+extern rct_map_element *g_currently_drawn_item;
+#else
+	#define g_currently_drawn_item RCT2_GLOBAL(0x9DE578, rct_map_element*)
```

Comment:
That's true, but I think in most cases it is. I can change it to `void *` if you want.